src/rprblender/export/image.py

```diff
@@ -94,9 +95,10 @@ def sync(rpr_context, image: bpy.types.Image, use_color_space=None, frame_number
 
     rpr_image.set_name(str(image_key))
 
-    # TODO: implement more correct support of image color space types
     # RPRImageTexture node color space names are in caps, unlike in Blender
-    if color_space in ('sRGB', 'BD16', 'Filmic Log', 'SRGB'):
+    if isinstance(rpr_context, context.RPRContext2):
```

Comment:
How does it work with images that should be used in "Linear"/"Raw" color space only - Normal maps, roughness maps, ets?

Reply:
fine.  but just like blender and cycles the user has to set that manually.